sites/fast-website/webpack.common.js

```diff
@@ -37,42 +36,40 @@ module.exports = {
             {
                 test: /\.css$/,
                 exclude: /node_modules/,
-                use: [
-                    {
-                        loader: MiniCssExtractPlugin.loader,
-                        options: {
-                            esModule: false,
-                        },
-                    },
-                    {
-                        loader: "css-loader",
-                    },
-                ],
+                loader: "css-loader",
             },
             {
                 test: /\.svg$/,
                 type: "asset/source",
             },
-            {
-                test: /\.ejs$/,
-                use: [
-                    {
-                        loader: "ejs-loader",
-                        options: {
-                            esModule: false,
-                        },
-                    },
-                ],
-            },
         ],
     },
     plugins: [
         new CleanWebpackPlugin(),
         new webpack.ProvidePlugin({ _: "lodash-es" }),
-        new HtmlWebpackPlugin({
-            title: "FAST",
-            manifest,
-            template: path.resolve(__dirname, "src/public/index.ejs"),
+        new HtmlBundlerPlugin({
+            entry: {
+                // define templates here
+                index: {
+                    // => dist/index.html
+                    import: "src/public/index.ejs",
+                    data: {
+                        title: "FAST",
+                        manifest,
+                    },
+                },
+            },
+            js: {
+                // output filename of extracted JS
+                filename: isDev ? "[name].js" : "bundle/[name].[contenthash:8].js",
+            },
+            css: {
+                // output filename of extracted CSS
+                filename: isDev ? "[name].css" : "bundle/[name].[contenthash:8].css",
+            },
+            loaderOptions: {
+                preprocessor: "ejs", // compile EJS templates to HTML
+            },
```

Comment:
The `entry` option has the same syntax as Webpack entry.
The key is a path to output HTML file without the `.html` extension.

You can define many pages in entry.
If you have no data for a template, you can use simple "one line" syntax, e.g.:

```js
entry: {
  "pages/about": "src/public/about.ejs", // output dist/pages/about.html
   // ...
}
```